packages/project-config/eslint-base.js

```diff
@@ -0,0 +1,41 @@
+// @ts-check
+
+/** @type {import('eslint').Linter.Config} */
+const config = {
+  root: true,
+  parser: '@typescript-eslint/parser',
+  plugins: ['@typescript-eslint/eslint-plugin'],
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:prettier/recommended',
+  ],
+  rules: {
+    '@typescript-eslint/no-empty-function': 'off',
+  },
```

Comment:
I want u to use `eslint-plugin-unused-imports` as well. Good shit. It automatically removes imports when used with `eslint --fix` in vscode settings.